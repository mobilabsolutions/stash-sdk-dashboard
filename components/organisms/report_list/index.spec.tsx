import ReportList from './index'
import { testRender, deepRender } from '../../../test_utils'

describe('Report list', () => {
  test('should show empty list', () => {
    testRender(ReportList, {
      reportList: []
    })
  })
  test('should display list of reports', () => {
    testRender(ReportList, {
      reportList: [
        {
          filterName: 'First report 10-12-2020'
        },
        {
          filterName: 'Second report 10-12-2020'
        },
        {
          filterName: 'Chargebacks report May'
        },
        {
          filterName: 'My default report'
        }
      ]
    })
  })
  test('should trigger download', () => {
    const downloadReport = jest.fn()
    const { getAllByText } = deepRender(ReportList, {
      reportList: [
        {
          filterName: 'First report 10-12-2020'
        },
        {
          filterName: 'Second report 10-12-2020'
        },
        {
          filterName: 'Chargebacks report May'
        }
      ],
      downloadReport
    })
    getAllByText('Download CSV')[1].click()
    expect(downloadReport).toBeCalledWith('Second report 10-12-2020')
  })
  test('should trigger deletion', () => {
    const deleteReport = jest.fn()
    const { getAllByText } = deepRender(ReportList, {
      reportList: [
        {
          filterName: 'First report 10-12-2020'
        },
        {
          filterName: 'Second report 10-12-2020'
        },
        {
          filterName: 'Chargebacks report May'
        }
      ],
      deleteReport
    })
    getAllByText('Delete Report')[1].click()
    expect(deleteReport).toBeCalledWith('Second report 10-12-2020')
  })
})
