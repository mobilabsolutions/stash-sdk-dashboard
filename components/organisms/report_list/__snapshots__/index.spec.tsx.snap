// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Report list should display list of reports 1`] = `
<div
  className="report_list__Wrapper-rx50gh-0 eMInVC"
>
  <div
    className="report_list__ReportItem-rx50gh-1 enwTcN"
  >
    <svg
      height={32}
      viewBox="0 0 32 32"
      width={32}
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        fill="none"
        fillRule="evenodd"
      >
        <circle
          cx="16"
          cy="16"
          fill="#B7D2FB"
          r="16"
        />
        <path
          d="M21.672 12.719c.219.219.328.479.328.781v9.375c0 .313-.11.578-.328.797a1.085 1.085 0 0 1-.797.328h-9.75c-.313 0-.578-.11-.797-.328a1.085 1.085 0 0 1-.328-.797V9.125c0-.313.11-.578.328-.797.219-.219.484-.328.797-.328h5.406c.157 0 .308.031.453.094.021.01.037.02.047.031.021 0 .042.01.063.031.094.052.172.11.234.172l4.344 4.39zm-1.422.156l-3.125-3.156v3.156h3.125zm.625 10V14h-3.75c-.313 0-.578-.11-.797-.328a1.085 1.085 0 0 1-.328-.797v-3.75h-4.875v13.75h9.75z"
          fill="#2E7EF2"
        />
      </g>
    </svg>
    <span
      className="report-title"
    >
      First report 10-12-2020
    </span>
    <span
      className="report-action"
    >
      <button
        className="flat__Button-sc-182ug78-0 jscbCh"
        onClick={[Function]}
      >
        <svg
          height={16}
          viewBox="0 0 16 16"
          width={16}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M7.59 12.857l-2.786-2.786a.614.614 0 0 1 0-.866.571.571 0 0 1 .428-.187c.167 0 .316.06.447.178L7.41 10.93V.625c0-.179.06-.327.178-.446a.614.614 0 0 1 .866 0 .591.591 0 0 1 .188.446v10.304l1.732-1.715a.56.56 0 0 1 .429-.196.6.6 0 0 1 .437.187.6.6 0 0 1 .188.438.56.56 0 0 1-.197.428l-2.786 2.786a.584.584 0 0 1-.428.179.584.584 0 0 1-.429-.179zm8.231-2.375a.584.584 0 0 1 .179.429v3.982c0 .297-.107.556-.321.777-.215.22-.477.33-.786.33H1.107c-.31 0-.571-.107-.786-.321A1.069 1.069 0 0 1 0 14.893V10.91c0-.167.06-.313.179-.438a.58.58 0 0 1 .437-.187.58.58 0 0 1 .438.187c.119.125.178.271.178.438v3.482a.363.363 0 0 0 .375.375h12.786a.363.363 0 0 0 .375-.375V10.91c0-.167.06-.313.178-.438a.58.58 0 0 1 .438-.187c.173 0 .318.065.437.196z"
            fill="#A3AAAF"
            fillRule="nonzero"
          />
        </svg>
         
        <span>
          Download CSV
        </span>
      </button>
      <button
        className="flat__Button-sc-182ug78-0 jscbCh"
        onClick={[Function]}
      >
        <svg
          height="16"
          viewBox="0 0 16 16"
          width="16"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M16 2.922a.585.585 0 0 1-.196.437.649.649 0 0 1-.463.187h-.357v11.1c0 .344-.124.638-.374.882-.25.243-.546.365-.89.365H2.458c-.345 0-.642-.122-.891-.365a1.188 1.188 0 0 1-.374-.882v-11.1H.64a.619.619 0 0 1-.454-.187A.608.608 0 0 1 0 2.913c0-.172.062-.32.187-.445a.619.619 0 0 1 .454-.187h3.297V1.247c0-.344.121-.638.365-.882.243-.243.543-.365.9-.365h5.594c.345 0 .642.122.891.365.25.244.374.538.374.882v1.034h3.279a.64.64 0 0 1 .659.641zM5.185 1.265v1.016h5.612V1.265H5.185zm8.534 13.38v-11.1H2.46v11.1h11.26zM8.428 5.88c.13.119.196.267.196.445v5.951c0 .178-.063.33-.187.455a.608.608 0 0 1-.446.187.608.608 0 0 1-.445-.187.619.619 0 0 1-.187-.455v-5.95c0-.179.062-.33.187-.455a.608.608 0 0 1 .445-.187c.172 0 .318.065.437.196zm-3.457 0c.13.119.196.267.196.445v5.951c0 .178-.062.33-.187.455a.619.619 0 0 1-.454.187.619.619 0 0 1-.455-.187.619.619 0 0 1-.187-.455v-5.95c0-.179.063-.33.187-.455a.619.619 0 0 1 .455-.187c.178 0 .326.065.445.196zm6.895 0c.131.119.196.267.196.445v5.951c0 .178-.062.33-.187.455a.619.619 0 0 1-.454.187.619.619 0 0 1-.454-.187.619.619 0 0 1-.187-.455v-5.95c0-.179.062-.33.187-.455a.619.619 0 0 1 .454-.187c.178 0 .327.065.445.196z"
            fill="#A3AAAF"
            fillRule="nonzero"
          />
        </svg>
         
        <span>
          Delete Report
        </span>
      </button>
    </span>
  </div>
  <div
    className="report_list__ReportItem-rx50gh-1 enwTcN"
  >
    <svg
      height={32}
      viewBox="0 0 32 32"
      width={32}
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        fill="none"
        fillRule="evenodd"
      >
        <circle
          cx="16"
          cy="16"
          fill="#B7D2FB"
          r="16"
        />
        <path
          d="M21.672 12.719c.219.219.328.479.328.781v9.375c0 .313-.11.578-.328.797a1.085 1.085 0 0 1-.797.328h-9.75c-.313 0-.578-.11-.797-.328a1.085 1.085 0 0 1-.328-.797V9.125c0-.313.11-.578.328-.797.219-.219.484-.328.797-.328h5.406c.157 0 .308.031.453.094.021.01.037.02.047.031.021 0 .042.01.063.031.094.052.172.11.234.172l4.344 4.39zm-1.422.156l-3.125-3.156v3.156h3.125zm.625 10V14h-3.75c-.313 0-.578-.11-.797-.328a1.085 1.085 0 0 1-.328-.797v-3.75h-4.875v13.75h9.75z"
          fill="#2E7EF2"
        />
      </g>
    </svg>
    <span
      className="report-title"
    >
      Second report 10-12-2020
    </span>
    <span
      className="report-action"
    >
      <button
        className="flat__Button-sc-182ug78-0 jscbCh"
        onClick={[Function]}
      >
        <svg
          height={16}
          viewBox="0 0 16 16"
          width={16}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M7.59 12.857l-2.786-2.786a.614.614 0 0 1 0-.866.571.571 0 0 1 .428-.187c.167 0 .316.06.447.178L7.41 10.93V.625c0-.179.06-.327.178-.446a.614.614 0 0 1 .866 0 .591.591 0 0 1 .188.446v10.304l1.732-1.715a.56.56 0 0 1 .429-.196.6.6 0 0 1 .437.187.6.6 0 0 1 .188.438.56.56 0 0 1-.197.428l-2.786 2.786a.584.584 0 0 1-.428.179.584.584 0 0 1-.429-.179zm8.231-2.375a.584.584 0 0 1 .179.429v3.982c0 .297-.107.556-.321.777-.215.22-.477.33-.786.33H1.107c-.31 0-.571-.107-.786-.321A1.069 1.069 0 0 1 0 14.893V10.91c0-.167.06-.313.179-.438a.58.58 0 0 1 .437-.187.58.58 0 0 1 .438.187c.119.125.178.271.178.438v3.482a.363.363 0 0 0 .375.375h12.786a.363.363 0 0 0 .375-.375V10.91c0-.167.06-.313.178-.438a.58.58 0 0 1 .438-.187c.173 0 .318.065.437.196z"
            fill="#A3AAAF"
            fillRule="nonzero"
          />
        </svg>
         
        <span>
          Download CSV
        </span>
      </button>
      <button
        className="flat__Button-sc-182ug78-0 jscbCh"
        onClick={[Function]}
      >
        <svg
          height="16"
          viewBox="0 0 16 16"
          width="16"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M16 2.922a.585.585 0 0 1-.196.437.649.649 0 0 1-.463.187h-.357v11.1c0 .344-.124.638-.374.882-.25.243-.546.365-.89.365H2.458c-.345 0-.642-.122-.891-.365a1.188 1.188 0 0 1-.374-.882v-11.1H.64a.619.619 0 0 1-.454-.187A.608.608 0 0 1 0 2.913c0-.172.062-.32.187-.445a.619.619 0 0 1 .454-.187h3.297V1.247c0-.344.121-.638.365-.882.243-.243.543-.365.9-.365h5.594c.345 0 .642.122.891.365.25.244.374.538.374.882v1.034h3.279a.64.64 0 0 1 .659.641zM5.185 1.265v1.016h5.612V1.265H5.185zm8.534 13.38v-11.1H2.46v11.1h11.26zM8.428 5.88c.13.119.196.267.196.445v5.951c0 .178-.063.33-.187.455a.608.608 0 0 1-.446.187.608.608 0 0 1-.445-.187.619.619 0 0 1-.187-.455v-5.95c0-.179.062-.33.187-.455a.608.608 0 0 1 .445-.187c.172 0 .318.065.437.196zm-3.457 0c.13.119.196.267.196.445v5.951c0 .178-.062.33-.187.455a.619.619 0 0 1-.454.187.619.619 0 0 1-.455-.187.619.619 0 0 1-.187-.455v-5.95c0-.179.063-.33.187-.455a.619.619 0 0 1 .455-.187c.178 0 .326.065.445.196zm6.895 0c.131.119.196.267.196.445v5.951c0 .178-.062.33-.187.455a.619.619 0 0 1-.454.187.619.619 0 0 1-.454-.187.619.619 0 0 1-.187-.455v-5.95c0-.179.062-.33.187-.455a.619.619 0 0 1 .454-.187c.178 0 .327.065.445.196z"
            fill="#A3AAAF"
            fillRule="nonzero"
          />
        </svg>
         
        <span>
          Delete Report
        </span>
      </button>
    </span>
  </div>
  <div
    className="report_list__ReportItem-rx50gh-1 enwTcN"
  >
    <svg
      height={32}
      viewBox="0 0 32 32"
      width={32}
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        fill="none"
        fillRule="evenodd"
      >
        <circle
          cx="16"
          cy="16"
          fill="#B7D2FB"
          r="16"
        />
        <path
          d="M21.672 12.719c.219.219.328.479.328.781v9.375c0 .313-.11.578-.328.797a1.085 1.085 0 0 1-.797.328h-9.75c-.313 0-.578-.11-.797-.328a1.085 1.085 0 0 1-.328-.797V9.125c0-.313.11-.578.328-.797.219-.219.484-.328.797-.328h5.406c.157 0 .308.031.453.094.021.01.037.02.047.031.021 0 .042.01.063.031.094.052.172.11.234.172l4.344 4.39zm-1.422.156l-3.125-3.156v3.156h3.125zm.625 10V14h-3.75c-.313 0-.578-.11-.797-.328a1.085 1.085 0 0 1-.328-.797v-3.75h-4.875v13.75h9.75z"
          fill="#2E7EF2"
        />
      </g>
    </svg>
    <span
      className="report-title"
    >
      Chargebacks report May
    </span>
    <span
      className="report-action"
    >
      <button
        className="flat__Button-sc-182ug78-0 jscbCh"
        onClick={[Function]}
      >
        <svg
          height={16}
          viewBox="0 0 16 16"
          width={16}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M7.59 12.857l-2.786-2.786a.614.614 0 0 1 0-.866.571.571 0 0 1 .428-.187c.167 0 .316.06.447.178L7.41 10.93V.625c0-.179.06-.327.178-.446a.614.614 0 0 1 .866 0 .591.591 0 0 1 .188.446v10.304l1.732-1.715a.56.56 0 0 1 .429-.196.6.6 0 0 1 .437.187.6.6 0 0 1 .188.438.56.56 0 0 1-.197.428l-2.786 2.786a.584.584 0 0 1-.428.179.584.584 0 0 1-.429-.179zm8.231-2.375a.584.584 0 0 1 .179.429v3.982c0 .297-.107.556-.321.777-.215.22-.477.33-.786.33H1.107c-.31 0-.571-.107-.786-.321A1.069 1.069 0 0 1 0 14.893V10.91c0-.167.06-.313.179-.438a.58.58 0 0 1 .437-.187.58.58 0 0 1 .438.187c.119.125.178.271.178.438v3.482a.363.363 0 0 0 .375.375h12.786a.363.363 0 0 0 .375-.375V10.91c0-.167.06-.313.178-.438a.58.58 0 0 1 .438-.187c.173 0 .318.065.437.196z"
            fill="#A3AAAF"
            fillRule="nonzero"
          />
        </svg>
         
        <span>
          Download CSV
        </span>
      </button>
      <button
        className="flat__Button-sc-182ug78-0 jscbCh"
        onClick={[Function]}
      >
        <svg
          height="16"
          viewBox="0 0 16 16"
          width="16"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M16 2.922a.585.585 0 0 1-.196.437.649.649 0 0 1-.463.187h-.357v11.1c0 .344-.124.638-.374.882-.25.243-.546.365-.89.365H2.458c-.345 0-.642-.122-.891-.365a1.188 1.188 0 0 1-.374-.882v-11.1H.64a.619.619 0 0 1-.454-.187A.608.608 0 0 1 0 2.913c0-.172.062-.32.187-.445a.619.619 0 0 1 .454-.187h3.297V1.247c0-.344.121-.638.365-.882.243-.243.543-.365.9-.365h5.594c.345 0 .642.122.891.365.25.244.374.538.374.882v1.034h3.279a.64.64 0 0 1 .659.641zM5.185 1.265v1.016h5.612V1.265H5.185zm8.534 13.38v-11.1H2.46v11.1h11.26zM8.428 5.88c.13.119.196.267.196.445v5.951c0 .178-.063.33-.187.455a.608.608 0 0 1-.446.187.608.608 0 0 1-.445-.187.619.619 0 0 1-.187-.455v-5.95c0-.179.062-.33.187-.455a.608.608 0 0 1 .445-.187c.172 0 .318.065.437.196zm-3.457 0c.13.119.196.267.196.445v5.951c0 .178-.062.33-.187.455a.619.619 0 0 1-.454.187.619.619 0 0 1-.455-.187.619.619 0 0 1-.187-.455v-5.95c0-.179.063-.33.187-.455a.619.619 0 0 1 .455-.187c.178 0 .326.065.445.196zm6.895 0c.131.119.196.267.196.445v5.951c0 .178-.062.33-.187.455a.619.619 0 0 1-.454.187.619.619 0 0 1-.454-.187.619.619 0 0 1-.187-.455v-5.95c0-.179.062-.33.187-.455a.619.619 0 0 1 .454-.187c.178 0 .327.065.445.196z"
            fill="#A3AAAF"
            fillRule="nonzero"
          />
        </svg>
         
        <span>
          Delete Report
        </span>
      </button>
    </span>
  </div>
  <div
    className="report_list__ReportItem-rx50gh-1 enwTcN"
  >
    <svg
      height={32}
      viewBox="0 0 32 32"
      width={32}
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        fill="none"
        fillRule="evenodd"
      >
        <circle
          cx="16"
          cy="16"
          fill="#B7D2FB"
          r="16"
        />
        <path
          d="M21.672 12.719c.219.219.328.479.328.781v9.375c0 .313-.11.578-.328.797a1.085 1.085 0 0 1-.797.328h-9.75c-.313 0-.578-.11-.797-.328a1.085 1.085 0 0 1-.328-.797V9.125c0-.313.11-.578.328-.797.219-.219.484-.328.797-.328h5.406c.157 0 .308.031.453.094.021.01.037.02.047.031.021 0 .042.01.063.031.094.052.172.11.234.172l4.344 4.39zm-1.422.156l-3.125-3.156v3.156h3.125zm.625 10V14h-3.75c-.313 0-.578-.11-.797-.328a1.085 1.085 0 0 1-.328-.797v-3.75h-4.875v13.75h9.75z"
          fill="#2E7EF2"
        />
      </g>
    </svg>
    <span
      className="report-title"
    >
      My default report
    </span>
    <span
      className="report-action"
    >
      <button
        className="flat__Button-sc-182ug78-0 jscbCh"
        onClick={[Function]}
      >
        <svg
          height={16}
          viewBox="0 0 16 16"
          width={16}
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M7.59 12.857l-2.786-2.786a.614.614 0 0 1 0-.866.571.571 0 0 1 .428-.187c.167 0 .316.06.447.178L7.41 10.93V.625c0-.179.06-.327.178-.446a.614.614 0 0 1 .866 0 .591.591 0 0 1 .188.446v10.304l1.732-1.715a.56.56 0 0 1 .429-.196.6.6 0 0 1 .437.187.6.6 0 0 1 .188.438.56.56 0 0 1-.197.428l-2.786 2.786a.584.584 0 0 1-.428.179.584.584 0 0 1-.429-.179zm8.231-2.375a.584.584 0 0 1 .179.429v3.982c0 .297-.107.556-.321.777-.215.22-.477.33-.786.33H1.107c-.31 0-.571-.107-.786-.321A1.069 1.069 0 0 1 0 14.893V10.91c0-.167.06-.313.179-.438a.58.58 0 0 1 .437-.187.58.58 0 0 1 .438.187c.119.125.178.271.178.438v3.482a.363.363 0 0 0 .375.375h12.786a.363.363 0 0 0 .375-.375V10.91c0-.167.06-.313.178-.438a.58.58 0 0 1 .438-.187c.173 0 .318.065.437.196z"
            fill="#A3AAAF"
            fillRule="nonzero"
          />
        </svg>
         
        <span>
          Download CSV
        </span>
      </button>
      <button
        className="flat__Button-sc-182ug78-0 jscbCh"
        onClick={[Function]}
      >
        <svg
          height="16"
          viewBox="0 0 16 16"
          width="16"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M16 2.922a.585.585 0 0 1-.196.437.649.649 0 0 1-.463.187h-.357v11.1c0 .344-.124.638-.374.882-.25.243-.546.365-.89.365H2.458c-.345 0-.642-.122-.891-.365a1.188 1.188 0 0 1-.374-.882v-11.1H.64a.619.619 0 0 1-.454-.187A.608.608 0 0 1 0 2.913c0-.172.062-.32.187-.445a.619.619 0 0 1 .454-.187h3.297V1.247c0-.344.121-.638.365-.882.243-.243.543-.365.9-.365h5.594c.345 0 .642.122.891.365.25.244.374.538.374.882v1.034h3.279a.64.64 0 0 1 .659.641zM5.185 1.265v1.016h5.612V1.265H5.185zm8.534 13.38v-11.1H2.46v11.1h11.26zM8.428 5.88c.13.119.196.267.196.445v5.951c0 .178-.063.33-.187.455a.608.608 0 0 1-.446.187.608.608 0 0 1-.445-.187.619.619 0 0 1-.187-.455v-5.95c0-.179.062-.33.187-.455a.608.608 0 0 1 .445-.187c.172 0 .318.065.437.196zm-3.457 0c.13.119.196.267.196.445v5.951c0 .178-.062.33-.187.455a.619.619 0 0 1-.454.187.619.619 0 0 1-.455-.187.619.619 0 0 1-.187-.455v-5.95c0-.179.063-.33.187-.455a.619.619 0 0 1 .455-.187c.178 0 .326.065.445.196zm6.895 0c.131.119.196.267.196.445v5.951c0 .178-.062.33-.187.455a.619.619 0 0 1-.454.187.619.619 0 0 1-.454-.187.619.619 0 0 1-.187-.455v-5.95c0-.179.062-.33.187-.455a.619.619 0 0 1 .454-.187c.178 0 .327.065.445.196z"
            fill="#A3AAAF"
            fillRule="nonzero"
          />
        </svg>
         
        <span>
          Delete Report
        </span>
      </button>
    </span>
  </div>
</div>
`;

exports[`Report list should show empty list 1`] = `
<div
  className="empty_data__Container-sc-19bkpdj-0 ezIMHh"
>
  <div
    className="internal"
  >
    <svg
      height={128}
      viewBox="0 0 311 128"
      width={311}
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        fill="none"
        fillRule="evenodd"
      >
        <g
          opacity=".495"
          transform="translate(94.233 33.713)"
        >
          <rect
            fill="#EDEFF0"
            height="61.112"
            rx="6.636"
            width="90.892"
          />
        </g>
        <path
          d="M234.474 127.825H84.027c-7.007 0-12.688-5.686-12.688-12.7 0-7.014 5.68-12.7 12.688-12.7H13.335c-7.007 0-12.688-5.686-12.688-12.7 0-7.014 5.68-12.7 12.688-12.7H85.84c7.007 0 12.688-5.686 12.688-12.7 0-7.014-5.68-12.7-12.688-12.7H40.525c-7.008 0-12.689-5.686-12.689-12.7 0-7.014 5.681-12.7 12.689-12.7h72.504c-7.008 0-12.688-5.686-12.688-12.7 0-7.014 5.68-12.7 12.688-12.7h177.636c7.007 0 12.688 5.686 12.688 12.7 0 7.014-5.68 12.7-12.688 12.7H218.16c7.008 0 12.688 5.686 12.688 12.7 0 7.014-5.68 12.7-12.688 12.7h39.878c7.007 0 12.688 5.686 12.688 12.7 0 7.014-5.68 12.7-12.688 12.7h-18.442c-8.835 0-15.998 5.686-15.998 12.7 0 4.676 3.625 8.909 10.876 12.7 7.007 0 12.688 5.686 12.688 12.7 0 7.014-5.68 12.7-12.688 12.7zm65.864-63.5c-5.71 0-10.338-4.637-10.338-10.357s4.628-10.357 10.338-10.357c5.709 0 10.337 4.637 10.337 10.357s-4.628 10.357-10.337 10.357z"
          fill="#A3AAAF"
          opacity=".123"
        />
        <rect
          fill="#D4D6E5"
          height="61.112"
          rx="6.636"
          transform="translate(94.922 40)"
          width="90.892"
          x=".01"
          y=".169"
        />
        <g
          transform="translate(109.514 49.023)"
        >
          <rect
            fill="#F6A094"
            height="61.112"
            rx="6.636"
            width="90.892"
          />
          <rect
            fill="#ED4E39"
            height="3.478"
            opacity=".498"
            rx="1.738"
            width="32.109"
            x="8.989"
            y="45.93"
          />
          <rect
            fill="#ED4E39"
            height="9.006"
            opacity=".498"
            rx="4.5"
            width="76.408"
            x="7.191"
            y="32.421"
          />
          <rect
            fill="#ED4E39"
            height="3.478"
            opacity=".498"
            rx="1.738"
            width="6.292"
            x="44.946"
            y="45.93"
          />
          <rect
            fill="#ED4E39"
            height="3.478"
            opacity=".498"
            rx="1.738"
            width="6.292"
            x="53.935"
            y="45.93"
          />
        </g>
        <g>
          <path
            d="M231.512 84.578l27.989 27.832a7.814 7.814 0 0 0 11.035-.016 7.766 7.766 0 0 0-.018-11.001l-32.36-32.145-6.663 6.663a6.117 6.117 0 0 0 .017 8.667z"
            fill="#4F5995"
          />
          <path
            d="M242.53 73.621l-11.018 10.957-4.359-4.325c-.018-.017 10.989-11.02 10.989-11.02l4.389 4.389z"
            fill="#131E61"
          />
          <path
            d="M192.747 65.937c-8.599-8.6-8.599-22.591 0-31.19 8.6-8.598 22.591-8.598 31.189 0 8.6 8.599 8.6 22.59 0 31.19-8.598 8.599-22.589 8.599-31.189 0zm49.947-1.502l-3.742-3.742c3.804-11.28 1.247-24.233-7.741-33.22-12.63-12.63-33.108-12.63-45.738 0-12.63 12.63-12.63 33.109 0 45.738 8.988 8.989 21.939 11.545 33.219 7.742l3.881 3.88 4.562-4.597c.018.017 11.023-10.987 11.023-10.987l-.117-.118 4.653-4.696z"
            fill="#4F5995"
          />
        </g>
      </g>
    </svg>
    <h1
      className="h1-sc-1ivb2ca-0 hnQWsF"
    >
      No reports created yet.
    </h1>
    <span>
      Please, generate a report first.
    </span>
  </div>
</div>
`;
